tests/src/cgeo/geocaching/LogEntryTest.java

```diff
@@ -0,0 +1,31 @@
+package cgeo.geocaching;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import cgeo.CGeoTestCase;
+import cgeo.geocaching.enumerations.LogType;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * LogEntry unit tests
+ */
+public class LogEntryTest extends CGeoTestCase {
+
+    public static void testComparator() {
+        final LogEntry logEntry1 = new LogEntry(100, LogType.FOUND_IT, "logEntry1 is older than logEntry2");
+        final LogEntry logEntry2 = new LogEntry(200, LogType.FOUND_IT, "logEntry2 is more recent than logEntry1");
+
+        final List<LogEntry> logList = new ArrayList<>(2);
+        logList.add(logEntry1);
+        logList.add(logEntry2);
+
+        Collections.sort(logList, LogEntry.DESCENDING_DATE_COMPARATOR);
+
+        assertThat(logList).hasSize(2);
```

Comment:
can be written as assertThat(logList).containsExactly(logEntry2, logEntry1). Does all 3 previous checks at once. :) I must admit I didn't use those more advanced assertions myself...